packages/lb-components/src/views/MainView/sidebar/PointCloudToolSidebar/components/annotatedBox/store.tsx

```diff
@@ -0,0 +1,27 @@
+import { create } from 'zustand';
```

Comment:
1.file desc
2.store can rebuild as a module(such as @/store/xxx),and other files can import